### ISSUE TITLE
CorHD dataset process program

### ISSUE THREAD

**Issue by jihai-cyber** [Sentient07/HeadHunter#9]
For the preprocessing .py program of the headhunter data set, you seem to have not uploaded it yet, can you please upload it? Thank you very much

**Comment by Sentient07** [comment]
Hi,
I don't understand your question. Can you please precisely mention what you're trying to do and what you're looking for in the existing code?

**Comment by jihai-cyber** [comment]
Hi,
Thanks for your reply.
I want to train a model for CorHD(HeadTracking21) dataset, but I don't see the create or preprocess program for this dataset.

**Comment by Sentient07** [comment]
What model do you want to train? If you'd like to reproduce the results of the paper, please refer to the README. If any part of it is unclear, feel free to mention that. Or, if you'd want to use it for a different purpose, please try to explain in detail so that I can understand your issue precisely, thank you.

**Comment by jihai-cyber** [comment]
HI，
I want to train a model for (c) Our proposed CroHD dataset. 
There are two questions about training

1. The following three files config/FT, config/PT, or config/SHead which config file is used to train this model.
2. In HeadHunter/head_detection/data/ file have two create a program -create_chuman.py,create_scuthead.py, but it doesn't have a program about CroHD dataset.

Thank you very much.

**Comment by Sentient07** [comment]
Hi,

1) --> IIRC config/FT is the correct file
2) --> Those files exist because you can use them to reproduce Table 2 using our released code. You don't need preprocessing prior to training for our dataset. Please refer [to README](https://github.com/Sentient07/HeadHunter#training) on how you can train

**Comment by jihai-cyber** [comment]
Hi,

config/FT need two options -train: 'HeadHunter_Train.txt' and valid: 'HeadHunter_Test.txt',andThere is no HeadHunter_Train.txt and 'HeadHunter_Test.txt in this website https://motchallenge.net/results/Head_Tracking_21/.

Thanks for your reply.

**Comment by Sentient07** [comment]
Hello,

As mentioned in the paper, we cannot release the Test set annotations. You can however create a validation split from the training set. Once you've downloaded the dataset, you can follow the instructions [here](https://github.com/Sentient07/HeadHunter#config-file) to create the respective `txt` files
